src/clinical_etl/mappings.py

```diff
@@ -79,22 +91,31 @@ def date_interval(data_values):
 
     Args:
         data_values: a values dict with a date
+        date_format: string with order of day, month and year in date (e.g. "DMY", "YMD", etc.)
```

Comment:
This isn't really an argument anymore...should this be here, or is it described somewhere else?